WooCommerce/Classes/ViewRelated/Products/Edit Product/DefaultProductFormTableViewModel.swift

```diff
@@ -33,14 +35,16 @@ struct DefaultProductFormTableViewModel: ProductFormTableViewModel {
          weightUnit: String? = ServiceLocator.shippingSettingsService.weightUnit,
          dimensionUnit: String? = ServiceLocator.shippingSettingsService.dimensionUnit,
          isDescriptionAIEnabled: Bool,
-         featureFlagService: FeatureFlagService = ServiceLocator.featureFlagService) {
+         featureFlagService: FeatureFlagService = ServiceLocator.featureFlagService,
+         stores: StoresManager = ServiceLocator.stores) {
         self.currency = currency
         self.currencyFormatter = currencyFormatter
         self.shippingValueLocalizer = shippingValueLocalizer
         self.weightUnit = weightUnit
         self.dimensionUnit = dimensionUnit
         self.isDescriptionAIEnabled = isDescriptionAIEnabled
         self.featureFlagService = featureFlagService
+        self.stores = stores
```

Comment:
Nit: I don't see `stores` being used, should we remove it?

Reply:
Good catch c2255b7178dfbc96033cf41c1dc73eacd6f5b04e